pkg/scalers/postgresql_scaler.go

```diff
@@ -74,6 +75,15 @@ func parsePostgreSQLMetadata(config *ScalerConfig) (*postgreSQLMetadata, error)
 		return nil, fmt.Errorf("no targetQueryValue given")
 	}
 
+	meta.activationTargetQueryValue = 0
```

Comment:
Could you add a unit test to check this default value?

Reply:
Query is calculated this way: `query: "SELECT CEIL(COUNT(*) / 5) FROM task_instance WHERE state='running' OR state='queued'"`

Reply:
is this needed? Parsing is tested and I haven't seen this kind of test implemented for other scalers?